### PR TITLE
Docs link & loader

### PR DESCRIPTION
**Pull request by maevalienard**
## Description
Docs link & loader + Props for each

## Demo
Example on: See message below 👇

### CODE REVIEW

File: packages/strapi-parts/src/Link/Link.stories.mdx [hunk 1]
```diff
@@ -10,11 +11,24 @@ import { Stack } from '../Stack';
...
 
-This is the doc of the `Link` component
+Links are used to send users to another page or an anchor. They are used for external navigation. 
 
-## Link
+**Best practices**
 
-Description...
+- Use TextButton component if the action is related to internal navigation.
+- Clicking on a Link opens a new tab.
```
Comment (mfrachet) [inline packages/strapi-parts/src/Link/Link.stories.mdx:19]: This is not always true. The "go back button" is a Link and it does not open a new tab 😊

File: packages/strapi-parts/src/Link/Link.stories.mdx [hunk 2]
```diff
@@ -10,11 +11,24 @@ import { Stack } from '../Stack';
...
 # Link
 
-This is the doc of the `Link` component
+Links are used to send users to another page or an anchor. They are used for external navigation. 
 
-## Link
+**Best practices**
 
-Description...
+- Use TextButton component if the action is related to internal navigation.
```
Comment (mfrachet) [inline packages/strapi-parts/src/Link/Link.stories.mdx:18]: Nope, a button is not a link

File: packages/strapi-parts/src/Link/Link.stories.mdx [hunk 3]
```diff
@@ -10,11 +11,24 @@ import { Stack } from '../Stack';
...
+- Clicking on a Link opens a new tab.
+- Links should always be used with an icon. If it is an external link, use the ExternalLink icon accordingly.
+- Links text should describe an action and start with a verb.
```
Comment (mfrachet) [inline packages/strapi-parts/src/Link/Link.stories.mdx:21]: they should describe the destination of the link and not start with a verb actually.

For instance, saying "Go back" does not provide detail about the navigation. What if the person was accessing the page using the URL? Does go back bring them to google?

My suggestion would be "Homepage", which is meaningful in more situations

File: packages/strapi-parts/src/Link/Link.stories.mdx [hunk 4]
```diff
@@ -10,11 +11,24 @@ import { Stack } from '../Stack';
...
+- Links text should be concise and describe their purpose. Users need to know exactly what they are about to do.
+- Use capital letters.
```
Comment (mfrachet) [inline packages/strapi-parts/src/Link/Link.stories.mdx:23]: Nope, if we want capital letters, we'll do it on the styling side, not in the code

File: packages/strapi-parts/src/Link/Link.stories.mdx [hunk 5]
```diff
@@ -10,11 +11,24 @@ import { Stack } from '../Stack';
...
+
+## Usage
+
+Links are used for external navigation. Each link should send to another page in a new tab. Moreover, they should be associated with an icon to indicate its external navigation purpose.
```
Comment (mfrachet) [inline packages/strapi-parts/src/Link/Link.stories.mdx:27]: See my first comment :)

File: packages/strapi-parts/src/Loader/Loader.stories.mdx [hunk 6]
```diff
@@ -10,14 +11,27 @@ import { Loader } from './Loader';
...
-This is the doc of the `Loader` component
+Loaders indicate some data is currently loading where content usually takes place. They help to notify users that loading is ongoing.
 
-## Loader
+**Best** **practices**
 
-Description...
+- Once the content appears, the loader disappears.
+- If the trigger only concerns a portion of a page, the loader should be placed in that part.
+- Loaders should be scaled down if they aimed to be placed in a small portion of a page.
```
Comment (mfrachet) [inline packages/strapi-parts/src/Loader/Loader.stories.mdx:20]: We don't have a spec for it nor a side to adjust. We can't "scale it on demand" :p

File: packages/strapi-parts/src/Loader/Loader.stories.mdx [hunk 7]
```diff
@@ -10,14 +11,27 @@ import { Loader } from './Loader';
...
+- Use a loader whenever the wait time is to be longer than 1 second.
```
Comment (mfrachet) [inline packages/strapi-parts/src/Loader/Loader.stories.mdx:21]: This will be tough until React 18 is out. Meaning replacing every single way strapi is getting data from the server (and that's a lot)

File: packages/strapi-parts/src/Loader/Loader.stories.mdx [hunk 8]
```diff
@@ -10,14 +11,27 @@ import { Loader } from './Loader';
...
+
+
+## Usage
+
+Even if loaders give an indication that an action has been processed, they do not provide details about what is happening on the back-end side.
```
Comment (mfrachet) [inline packages/strapi-parts/src/Loader/Loader.stories.mdx:26]: I personally think this is not a valid rule. As a user, I want to know that my locales are resolving (and not my permissions somehow). Loading should be contextualized, to me

**Review by mfrachet — CHANGES_REQUESTED** [review]
A link is not a button 😝

File: packages/strapi-parts/src/Link/Link.stories.mdx (hunk 4 above)
Comment (HichamELBSI) [reply packages/strapi-parts/src/Link/Link.stories.mdx:23]: In the new admin views, sometimes, there is no capital letters 

Screenshot 2021-08-17 at 09 55 12

**Review by HichamELBSI — CHANGES_REQUESTED** [review]

File: packages/strapi-parts/src/Link/Link.stories.mdx (hunk 1 above)
Comment (maevalienard) [reply packages/strapi-parts/src/Link/Link.stories.mdx:19]: Well this is a mistake coming from the design side then. In an ideal world, this should be like mentioned in the documentation

File: packages/strapi-parts/src/Link/Link.stories.mdx (hunk 2 above)
Comment (maevalienard) [reply packages/strapi-parts/src/Link/Link.stories.mdx:18]: You're right. Maybe I could rework the sentence by saying something like "Use TextButton component if an action needs to be triggered within the admin panel"?

File: packages/strapi-parts/src/Link/Link.stories.mdx (hunk 3 above)
Comment (maevalienard) [reply packages/strapi-parts/src/Link/Link.stories.mdx:21]: That's another topic we could discuss later on 😅 I'll remove this sentence

File: packages/strapi-parts/src/Link/Link.stories.mdx (hunk 4 above)
Comment (maevalienard) [reply packages/strapi-parts/src/Link/Link.stories.mdx:23]: Hicham: The idea is to have consistency, so if it's not capital letter here, this is a mistake coming from the design side 😅 
Marvin: We don't do it yet on the styling side?

File: packages/strapi-parts/src/Loader/Loader.stories.mdx (hunk 6 above)
Comment (maevalienard) [reply packages/strapi-parts/src/Loader/Loader.stories.mdx:20]: Alright, I thought something "easy" to do, let's remove it then

File: packages/strapi-parts/src/Loader/Loader.stories.mdx (hunk 7 above)
Comment (maevalienard) [reply packages/strapi-parts/src/Loader/Loader.stories.mdx:21]: Let's remove it for now then :)

File: packages/strapi-parts/src/Loader/Loader.stories.mdx (hunk 8 above)
Comment (maevalienard) [reply packages/strapi-parts/src/Loader/Loader.stories.mdx:26]: But it's not the case, right?

File: packages/strapi-parts/src/Link/Link.stories.mdx (hunk 1 above)
Comment (mfrachet) [reply packages/strapi-parts/src/Link/Link.stories.mdx:19]: I'm not sure to understand what you mean

File: packages/strapi-parts/src/Link/Link.stories.mdx (hunk 2 above)
Comment (mfrachet) [reply packages/strapi-parts/src/Link/Link.stories.mdx:18]: Yes, you can take the example of opening a modal I think 😊

File: packages/strapi-parts/src/Link/Link.stories.mdx (hunk 4 above)
Comment (mfrachet) [reply packages/strapi-parts/src/Link/Link.stories.mdx:23]: What I mean is that if we want to transform the text to be in uppercase, it won't be the job of the developer to type things in uppercase. Imagine a blind user navigating and be yelled on the face saying "LOG IN VIA STRAPI YOU COWARD" 😛 .

When done on the design side (I mean, on the CSS part of the thing, automatically, without a developer action), the text will be announced less agressivel :).

TL;DR => people just write "Log in via Strapi" and we transform the text for them on the according component

File: packages/strapi-parts/src/Loader/Loader.stories.mdx (hunk 6 above)
Comment (mfrachet) [reply packages/strapi-parts/src/Loader/Loader.stories.mdx:20]: And I won't bring the consistency argument with you 😝

File: packages/strapi-parts/src/Loader/Loader.stories.mdx (hunk 8 above)
Comment (mfrachet) [reply packages/strapi-parts/src/Loader/Loader.stories.mdx:26]: It's the case, but it's hidden. In this scenario, people using assistive technologies have more feedbacks than people using a screen monitor 😊 . But as mentioned, that's a personal take. But still, a label should be provided, even if hidden, for people using ATs

File: packages/strapi-parts/src/Link/Link.stories.mdx (hunk 4 above)
Comment (maevalienard) [reply packages/strapi-parts/src/Link/Link.stories.mdx:23]: @mfrachet If you look at the UI documentation, you'll see that the Link component is supposed to have a Capital letter style. I think Hicham is not talking about the button, but the "Forgot your password" below. I mentioned that if it's not capital letter, then it's a design mistake :) (if the semantic is fine, which is think so? 🤔 )

File: packages/strapi-parts/src/Loader/Loader.stories.mdx (hunk 8 above)
Comment (maevalienard) [reply packages/strapi-parts/src/Loader/Loader.stories.mdx:26]: @mfrachet Let's talk about this directly, so we can make a decision quickly :)

Comment (HichamELBSI) [reply packages/strapi-parts/src/Loader/Loader.stories.mdx:24]: @maevalienard Has a decision been made?

Comment (maevalienard) [reply packages/strapi-parts/src/Loader/Loader.stories.mdx:24]: Not yet 😂 Let me bother you instead :p

**Review by HichamELBSI — APPROVED** [review]
LGTM thanks
